pod/perlsub.pod

```diff
@@ -108,7 +108,7 @@ one large indistinguishable list.
 
 If no C<return> is found and if the last statement is an expression, its
 value is returned.  If the last statement is a loop control structure
-like a C<foreach> or a C<while>, the returned value is unspecified.  The
+like a L<C<foreach>|perlfunc/foreach> or a L<C<while>|perlfunc/while>, the returned value is unspecified.  The
```

Comment:
These should link to `perlsyn/Compound Statements` as the perlfunc entry is a dummy.